operations/identitydocuments.md

```diff
@@ -0,0 +1,331 @@
+
+### Request
+
+`[PlatformAddress]/api/connector/v1/identityDocuments/add`
+
+```javascript
+{
+  "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+  "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+  "Client": "Sample Client 1.0.0",
+  "IdentityDocuments": [
+    {
+      "CustomerId": "e8a72a69-c20b-4278-b699-ab0400a32ecc",
+      "Type": "Passport",
+      "Number": "123456789",
+      "ExpirationDate": "2040-11-20T00:00:00Z",
+      "IssuanceDate": "2020-11-20T00:00:00Z",
+      "IssuingCountryCode": "CZ",
+      "IssuingCity": "Prague",
+      "IdentityDocumentSupportNumber": "S-123456"
+    }
+  ],
+  "ChainId": "3fa85f64-5717-4562-b3fc-2c963f66afa6"
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `Client` | string | required | Name and version of the client application. |
+| `ChainId` | string | optional | Unique identifier of the chain. Required when using [Portfolio Access Tokens](../guidelines/multi-property.md), ignored otherwise. |
+| `IdentityDocuments` | array of [Identity documents parameters](identitydocuments.md#identity-documents-parameters) | required, max 10 items | Identity documents to be added |
+
+#### Identity documents parameters
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `CustomerId` | string | required | Identifier of the `Customer`. |
+| `Type` | [Document type](reservations.md#document-type) | required | Type of the document. |
+| `Number` | string | required | Number of the document (e.g. passport number). |
+| `ExpirationDate` | string | optional | Expiration date in ISO 8601 format. |
+| `IssuanceDate` | string | optional | Date of issuance in ISO 8601 format. |
+| `IssuingCountryCode` | string | optional | ISO 3166-1 code of the `Country`). |
+| `IssuingCity` | string | optional | City where the document was issued. |
+| `IdentityDocumentSupportNumber` | string | optional | Identity document support number. Only required for Spanish identity cards in Spanish hotels. |
+
+### Response
+
+```javascript
+{
+  "IdentityDocuments": [
+    {
+      "Id": "e8a72a69-c20b-4278-b699-ab0400a32ecc",
+      "CustomerId": "fadd5bb6-b428-45d5-94f8-fd0d89fece6d",
+      "Type": "Passport",
+      "Number": "M1234567",
+      "ExpirationDate": "2040-10-24T00:00:00Z",
+      "IssuanceDate": "2024-10-24T00:00:00Z",
+      "IssuingCountryCode": "CZ",
+      "IssuingCity": "Prague",
+      "IdentityDocumentSupportNumber": "S-123456"
+    }
+  ]
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `IdentityDocuments` | array of [Identity document (ver 2024-10-25)](identitydocuments.md#identity-document-ver-2024-10-25) | required, max 10 items | Updated identity documents of customers. |
```

Comment:
```suggestion
| `IdentityDocuments` | array of [Identity document (ver 2024-10-25)](identitydocuments.md#identity-document-ver-2024-10-25) | required, max 10 items | Added identity documents. |
```

Reply:
This is shared between update and add, so I will change the wording here to "Modified"

Reply:
Ah, I see. So the description needs to be made independent of the operation that uses it, e.g. "identity documents affected by the operation" or perhaps simply "identity documents", so the object can be re-used.